tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+# role tasks
+- name: Upgrade kexec-tools to latest
+  yum:
+    name: kexec-tools
+    state: latest
+  become: true
+  become_user: "{{ kdump_become_user }}"
+
+- name: Waive previous nfs setup if enabled
+  lineinfile: dest="{{ kdump_conf_path }}"
```

Comment:
When writing Ansible YAML we should always colons instead of equals signs, with two-space indentation.  Example:
```
lineinfile: 
  dest: "{{ kdump_conf_path }}"
  state: present
```